test/feed.test.js

```diff
@@ -88,7 +88,7 @@ Object.keys(testAPIs).forEach(API => {
 
       it('adds an item that is > 256 bytes', async () => {
         db = await orbitdb1.feed('third')
-        let msg = new Buffer(1024)
+        let msg = Buffer.alloc(1024)
```

Comment:
`new Buffer()` is deprecated in Node.js 10, changing these to the current syntax.